android/src/com/mapswithme/util/UiUtils.java

```diff
@@ -227,9 +227,10 @@ public static void visibleIf(boolean condition, View... views)
       invisible(views);
   }
 
-  public static void showIf(boolean condition, View view)
-  {
-    view.setVisibility(condition ? View.VISIBLE : View.GONE);
+  public static void showIf(boolean condition, View view) {
+    if (view != null) {
```

Comment:
This fix is a dirty hack. Please find and fix the root cause. Why someone wants to show a widget which doesn't exist? I guess that we removed some widgets from layouts, but forget to remove usage in the code.